src/picongpu/include/plugins/output/images/Visualisation.hpp

```diff
@@ -474,7 +474,7 @@ ALPAKA_FN_ACC void operator()(
         "KernelDivideAnyCell has to be used as a 1 dimensional kernel only!");
 
     DataSpace<1> const blockSize(alpaka::workdiv::getWorkDiv<alpaka::Block, alpaka::Threads>(acc));
-    DataSpace<1> const blockIndex(alpaka::workdiv::getWorkDiv<alpaka::Grid, alpaka::Blocks>(acc));
+    DataSpace<1> const blockIndex(alpaka::idx::getIdx<alpaka::Grid, alpaka::Blocks>(acc));
```

Comment:
We can not use the CUDA blockIdx name within a kernel because this would result in a compiler error when building for CUDA ;-)
Maybe this should be documented somewhere in an alpaka introduction because this is one of the traps beginners coming from CUDA will be tempted to fall into.